### PR TITLE
expression, parser: refine parser of localtime and localtimestamp

### PR DESCRIPTION
**Pull request by XuHuaiyu**
localtime/ localtime()/ localtimestamp/ localtimestmp() 
are all the synonyms of NOW().

PTAL @zz-jason @breeswish

### CODE REVIEW

File: expression/integration_test.go [hunk 1]
```diff
@@ -1504,6 +1504,10 @@ func (s *testIntegrationSuite) TestTimeBuiltin(c *C) {
 	// for extract
 	result = tk.MustQuery(`select extract(day from '800:12:12'), extract(hour from '800:12:12'), extract(month from 20170101), extract(day_second from '2017-01-01 12:12:12')`)
 	result.Check(testkit.Rows("12 800 1 1121212"))
+
+	// for localtime, localtimestamp
+	result = tk.MustQuery(`select localtime() = now(), localtime = now(), localtimestamp() = now(), localtimestamp = now()`)
```
Comment (zz-jason) [inline expression/integration_test.go]:
```sql
select current_timestamp = now();
```
should also return 1, maybe we can add it into this test

Comment (winkyao) [reply expression/integration_test.go]: +1

**Comment by XuHuaiyu** [comment]
PTAL @zz-jason @winkyao

**Pull request merged**